packages/docs/src/page-configs/ui-elements/navbar/examples/Height.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
<img width="629" alt="Screenshot 2022-11-24 at 14 57 02" src="https://user-images.githubusercontent.com/46869767/203778867-f02c4956-ee7e-4ad6-9220-a4605036aa9f.png">
Do we really need this example? Because it's about inline styles not about component props. Better ask @asvae :)

Reply:
Or maybe add some description here